classes/publication/PKPPublication.inc.php

```diff
@@ -25,27 +25,32 @@ class PKPPublication extends DataObject {
 	 * - the publication's primary locale
 	 * - the first locale we find data for
 	 *
-	 * @param string $key
-	 * @param string $preferredLocale
+	 * @param $key string
+	 * @param $preferredLocale string
+	 * @param $selectedLocale Optional reference to receive locale used for return value.
 	 * @return mixed
 	 */
-	public function getLocalizedData($key, $preferredLocale = null) {
+	public function getLocalizedData($key, $preferredLocale = null, &$selectedLocale = null) {
```

Comment:
Is this going to create PHP notices because the function signature doesn't match the parent class?

Reply:
No, we're fine there -- the two are compatible (i.e. calls based on the parent's signature don't violate this signature).